### DISCUSSION TITLE
Request: Character Gestures

### DISCUSSION THREAD

**Discussion by gilbigdog** [rxhanson/Multitouch-Community#104] [Ideas]
It was one of my favorit features in JiTouch which is no longer working. I can draw a gesture using forefinger and ringfinger(there should be enough space between the fingers otherwise, it was recognized as just two finger event). 

It was useful in maximizing(Draw a straight line from Left bottom to right top)/minimizing windows and launching applications.(like drwaing "C" launches "Chrome").

**Comment by rxhanson** [comment]
This has been asked for in the past by JiTouch users. I don't have plans to add this in (sorry). I was planning on adding in one touch in a line from each of the corners, but in practice I didn't like them and I ended up creating the rest 3/4/5 and remove all but one gestures, which were much more ergonomic when tested out. I might still add the corner line ones, but it would be a matter of prioritizing that over other features.
